portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase

```diff
@@ -29,6 +29,15 @@ definition {
 		ACUtils.tearDownAC();
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8462 | Test Summary: Inform the User when there are no disabled Segments"
+	@ignore = "true"
+	@priority = "5"
+	test AssertUserIsInformedWhenThereAreNoDisabledSegments {
+
+		// TODO LRAC-8462 AssertUserIsInformedWhenThereAreNoDisabledSegments
```

Comment:
hi @AfonsoHSantos , this test is a priority should be 3 according to our test map.